staging/src/k8s.io/kubectl/pkg/describe/versioned/describe.go

```diff
@@ -2350,8 +2350,14 @@ func (i *IngressDescriber) Describe(namespace, name string, describerSettings de
 }
 
 func (i *IngressDescriber) describeBackend(ns string, backend *networkingv1beta1.IngressBackend) string {
-	endpoints, _ := i.CoreV1().Endpoints(ns).Get(context.TODO(), backend.ServiceName, metav1.GetOptions{})
-	service, _ := i.CoreV1().Services(ns).Get(context.TODO(), backend.ServiceName, metav1.GetOptions{})
+	endpoints, err := i.CoreV1().Endpoints(ns).Get(context.TODO(), backend.ServiceName, metav1.GetOptions{})
+	if err != nil {
+		return fmt.Sprintf("<none> (failed to fetch endpoints: %v)", err)
+	}
+	service, err := i.CoreV1().Services(ns).Get(context.TODO(), backend.ServiceName, metav1.GetOptions{})
+	if err != nil {
+		return fmt.Sprintf("<none> (failed to fetch services: %v)", err)
+	}
```

Comment:
I'd prefer we leave the old style, this is unnecessarily messing the output and would be a precedent. See for example ServiceDescriber where it's fine to ignore error, since this is only a temporary issue, usually when describing a resource.

Reply:
I don't know what the precedent is, but I think that's really needed. As a person sometimes debugging cluster for a living, I really hate when I receive invalid information (even if it's temporary), it can really mess up my investigation. I agree that it's a little verbose, but I think that's useful.

Reply:
I agree with your take on this, but I'd rather have a consistent way of stating this, and make it work everywhere. Honestly I'm inclined to error out entirely rather than show only partial/incorrect data. But let's keep it out of this PR.